yt_dlp/downloader/fragment.py

```diff
@@ -347,7 +359,44 @@ def decrypt_fragment(fragment, frag_content):
 
         return decrypt_fragment
 
-    def download_and_append_fragments(self, ctx, fragments, info_dict, *, pack_func=None, finish_func=None):
+    def download_and_append_fragments_multiple(self, *args, pack_func=None, finish_func=None):
```

Comment:
I don't like this function name, but I dont have any better ideas either 😅

Reply:
the next candidate I came up with is `parallel_download_and_append_fragments`. how about this?

Reply:
`download_and_append_fragments` is also parallel (if `-N` is set) then :sweat_smile: